src/events/messageCreate.ts

```diff
@@ -24,6 +30,7 @@ export const handleNormalMessage = async (message: Message) => {
     channel_id: message.channelId,
     timestamp: message.createdAt.toISOString(),
     content: message.content,
+    sticker_items: stickers,
```

Comment:
```suggestion
    sticker_items: stickers.map(s => s.id),
```
pass object `Sticker` vào nó không serialize được sẽ dính error